src/transcript/TranscriptEntity.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default class TranscriptEntity {
```

Comment:
Do we need to update demo app to demonstrate how to use the additional data?

Reply:
Yes, the demo app has the changes and it has been tested locally. But since demo app needs AWS SDK to start the transcription, the changes has been tested using curl command since AWS SDK work is still in progress. The demo app can be used to demonstrate the PII/PHI data that are identified using some color coding and seeing the redacted values. The confidence demonstration is still under discussion.

Reply:
>  The demo app can be used to demonstrate the PII/PHI data that are identified using some color coding and seeing the redacted values.

Are we going to have a separate CR with these changes or should we include them in this CR?

Reply:
We will include the demo app changes in the same PR.

Reply:
The demo app has been included in this PR